src/util/submit-addon.js

```diff
@@ -237,7 +258,7 @@ export default class Client {
   }
 
   async fetch(url, method = 'GET', body) {
-    log.info(`Fetching URL: ${url.href}`);
+    log.info(`${method}ing URL: ${url.href}`);
```

Comment:
Nit: "GETing URL"?

```suggestion
    log.info(`Fetching URL: ${method} ${url.href}`);
```

Reply:
It was a deliberate  - I quite liked `GETing URL: https://...` and `PATCHing URL: https://...` - the verbs read well as, well, verbs in a sentence.  ("CLIMBing the mountain: Everest" vs "Doing the mountain: CLIMB Everest")

But I'm not committed to it if you think keeping "Fetching" in the text is better.